pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/EmptyResponseIntegrationTest.java

```diff
@@ -260,6 +261,56 @@ public void testSchemaFallbackAggregatedFields() throws Exception {
     assertDataTypes(response, "LONG", "DOUBLE");
   }
 
+  @Test(priority = 1)
```

Comment:
is `priority = 1` a new thing?

Reply:
Usually is not a good practice to specify priorities in tests. Is there a reason why we need to use them here?